index.html

```diff
@@ -941,11 +963,211 @@ <h2>Specifying the Type</h2>
 <h1>Advanced Concepts</h1>
 
 <p>JSON-LD has a number of features that provide functionality above and beyond
-  the core functionality described above. The following section describes this
+  the core functionality described above. JSON can be used to express data
+  using a variety of different stuctures, and the features described in this
+  section can be used to interpret a variety of different JSON structures into
```

Comment:
"to interpret a variety of different JSON structures" -> "to interpret those structures"